siteorigin-panels.php

```diff
@@ -719,12 +720,19 @@ function siteorigin_panels_generate_css($post_id, $panels_data = false){
 			}
 		}
 
-		// Add the bottom margin to any grids that aren't the last
-		if($gi != count($panels_data['grids'])-1){
+		if($panels_margin_bottom_last_row) {
```

Comment:
You could accomplish the same thing by just adding ` || !empty( $panels_margin_bottom_last_row )` to the original conditional expression. So in future, if anything needs to change about how we're calling `$css->add_row_css` we only have to change it in one place.